person.rb

```diff
@@ -1,4 +1,6 @@
-class Person
+require_relative './interface'
+
+class Person < Nameable
```

Comment:
- [ ] The `Person` class should have a method `correct_name` that return `name`. Please add it.